packages/neuron-ui/src/components/NervosDAO/index.tsx

```diff
@@ -98,6 +99,9 @@ const NervosDAO = ({
   useEffect(() => {
     updateNervosDaoData({ walletID: wallet.id })(dispatch)
     updateDepositValue(`${MIN_DEPOSIT_AMOUNT}`)
+    getBlockByNumber('0x0')
```

Comment:
Why not calling `get_header_by_number` to only fetch the genesis block header, as the block is big and slow to fetch?

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/1119/commits/b8d44731887efed8042469265abdbad5d671ac27